src/include/index.h

```diff
@@ -201,8 +201,21 @@ const char MAIN_page[] PROGMEM = R"=====(
             <p>ID must be four characters long. It can be any capital letters of the Latin alphabet A-Z except O, or any
                 numbers 1-9.</p>
             <label for="status">Your message:</label><br/>
-            <textarea class="textbox comments textbox-full" name="message" id="commentsInput" cols="30"
-                      rows="2"></textarea>
+            <textarea class="textbox comments textbox-full" maxlength=200 name="message" id="commentsInput" cols="30"
```

Comment:
Keep in mind that this is only a front-end limitation. Whatever handles this form field on the back end needs to enforce the limit as well.

Reply:
good point! Right now the data packet is 229 bytes long. As a default, we put the max characters, but maybe make the length configurable.